pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -270,6 +270,10 @@ func (o *MustGatherOptions) Validate() error {
 	if len(o.NodeName) > 0 && strings.ContainsAny(o.NodeName, "/%") {
 		return fmt.Errorf("--node-name may not contain '/' or '%%'")
 	}
+	destDirRe := regexp.MustCompile(":")
+	if destDirRe.MatchString(o.DestDir) {
+		return fmt.Errorf("--dest-dir may contain special characters such as colon(:)")
+	}
```

Comment:
I think we should use `strings.Contains` here instead of `regexp` as it's far lighter/cheaper. Also, the error message appears to be missing the word `not`. Should be `--dest-dir may NOT contain special characters...".

Reply:
@deejross thanks, done